Sources/ApolloCodegenLib/ApolloCodegenOptions.swift

```diff
@@ -58,7 +58,6 @@ public struct ApolloCodegenOptions {
   }
   
   let codegenEngine: CodeGenerationEngine
-  let additionalInflectionRules: [InflectionRule]
```

Comment:
just please please please remember to re-add this, it solves so many goddamned problems